From a393162f6b7f6f9c246b7b198713bfb3ad71ef89 Mon Sep 17 00:00:00 2001
From: Dave Eddy <dave@daveeddy.com>
Date: Thu, 30 May 2019 14:31:08 -0400
Subject: [PATCH] MANTA-4303 electric-boray fails if the optional "fast" config
 is not present Reviewed by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
 Approved by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>

---
 lib/server.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/server.js b/lib/server.js
index d98496d..41792ba 100644
--- a/lib/server.js
+++ b/lib/server.js
@@ -221,6 +221,7 @@ LimitOffsetStream.prototype.getNextRecord =
 function createServer(options, callback) {
     assert.object(options, 'options');
     assert.optionalObject(options.fast, 'options.fast');
+    options.fast = options.fast || {};
     assert.optionalNumber(options.fast.client_crc_mode,
         'options.fast.client_crc_mode');
     assert.optionalNumber(options.fast.server_crc_mode,
-- 
2.21.0

